src/_data/capability-matrix.yml

```diff
@@ -176,6 +198,10 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: Implemented by merging side input as a normal stream in Gearpump
+          - class: mapreduce
+            l1: 'Yes'
+            l2: fully supported
+            l3: ''
```

Comment:
Normally, I'd want one sentence about the MapReduce feature used to implement them, but I think with the upcoming revamp it is not worth filling it out. We should have this page revised before the runner is on master.

Reply:
ack